conf/ocsci/submariner_downstream.yaml

```diff
@@ -1,2 +1,3 @@
 ENV_DATA:
   submariner_source: "downstream"
+  subctl_version: "subctl-rhel8:v0.16"
```

Comment:
as we are still running on CentOS stream 8 , this might cause the compatibility issues to run on RHEL 8 based system as we recently saw with glibc issue.

We are going to move in matter of weeks to CentOS stream9 , but cannot say when exactly.

It can run without an issue, but might not.

Reply:
tried this on centos stream rlease 9 and its working